tests/phpunit/TestFramework/PhpUnit/Config/XmlConfigurationHelperTest.php

```diff
@@ -486,9 +489,12 @@ public function schemaProvider(): \Generator
 
     /**
      * @dataProvider schemaProvider
+     *
+     * @group integration
```

Comment:
to be added in a separate PR; apply same comment as above in the phpdoc (dependency on network)